### ISSUE TITLE
[Feature Request] Handling multi activities routes

### ISSUE THREAD

**Issue by pyrog** [vmarc/knooppuntnet#214]
Some hiking routes are shared with at least MTB and/or riding.

Current OSM tagging scheme support that: it use multi values in the `network` key (separated by `;`)
Example:
[Les trois Monts](https://www.openstreetmap.org/relation/13364852) a hiking and MTB route on top of the [Grand Dole node network](https://knooppuntnet.nl/en/analysis/network/13305300).
```
type=route
route=hiking;mtb

```
This route is displayed in [hiking](https://hiking.waymarkedtrails.org/#route?id=13364852&map=13!47.1113!5.4374) and [MTB](https://mtb.waymarkedtrails.org/#route?id=13364852&map=13!47.1113!5.4374) WaymarkedTrails.

I [changed](https://knooppuntnet.nl/en/analysis/changeset/112913553/4771601) the `route` tag of all shared routes of the node network, but this change raise a new fact:

**RouteTagInvalid**

Invalid value in required tag "route" in route relation. A bicycle route relation needs to have value "bicycle" in its "route" tag. A hiking route relation needs to have one of the following values in its "route" tag: "foot", "hiking", or "walking". Note that "walking" is a value that is frequently found, but not actually documented as a valid value in the OSM wiki pages. Other potential values are "horse", "motorboat", "canoe" and "inlineskates"*.

**Note:** There is **510** multi activities routes according [overpass](https://overpass-turbo.eu/s/1clT) and around **410** routes according  [TagInfo](https://taginfo.openstreetmap.org/keys/route#values) (type ; in the search value and count manually good values)

See also #213

**Comment by vmarc** [comment]
The analysis logic has been changed so that _RouteTagInvalid_ is no longer generated when the _"route"_ tag contains multiple values separated by semicolon(s). Support for MTB routes is still a separate issue.

**Comment by pyrog** [comment]
Thank you 😄 

Note: the facts are still visible in the [history](https://experimental.knooppuntnet.nl/en/analysis/route/13325909/changes) in version 2 and on the map a low zoom levels
(I think the red colour will change to green in the next tiles generation)

<img width="33%" alt="Map" src="https://user-images.githubusercontent.com/2158081/139043365-d8f104a3-bb5b-4d13-8466-0be06ba30486.png"> <img width="33%" alt="Facts" src="https://user-images.githubusercontent.com/2158081/139043628-1bf2e9eb-5667-4884-ba36-372f39c52e10.png">

**Comment by vmarc** [comment]
> Note: the facts are still visible in the history in version 2

The fact will remain visible in the history until release v4.0 where the history of all nodes/routes/networks will be completely recalculated.

> I think the red colour will change to green in the next tiles generation

Good point. Normally tagging changes processing results in immediate tile updates, but software updates do not trigger tile updates. Without action all tiles would be recalculated overnight (03:00h). I have now manually triggered tile recalculation, so the red should disappear during the  next 20 minutes or so.

**Comment by pyrog** [comment]
Routes are green now, thanks 😄

**Comment by pyrog** [comment]
I checked the hiking and riding route [Ctre équestre la Sauline - Les Près Gaulard](https://experimental.knooppuntnet.nl/en/analysis/route/13311165/changes)

There is no fact and it is displayed on the hiking map 😄 
But the route is not rendered on the riding map 😢 
<img width="33%" alt="Hiking" src="https://user-images.githubusercontent.com/2158081/139066704-a68ccbe5-bb3d-4bcf-9039-918fdda47721.png"> <img width="33%" alt="Riding" src="https://user-images.githubusercontent.com/2158081/139066763-6764bbed-fa32-4e21-b7bc-4005b5ce4119.png">

**Comment by vmarc** [comment]
> But the route is not rendered on the riding map

Yes, this is because the analysis at this moment is still network based, and the route is now analyzed in the context of network "Réseau pédestre du Grand Dole" which is a hiking network. I think the  analysis would not handle it well if there also was a riding network that contained that route.  Analysis logic will change drastically (route based rather than network based) with the next release v4.0. I will try to make it work in that release.

**Comment by pyrog** [comment]
**Note:** multivalued tags are not fine (i.e. `route=hiking;mtb`)
- At least JOSM don't display this as a route in selection or relations lists.
- Overpass query are difficult and slow (need regular expression)

Using tags like `hiking=yes`, `mtb=yes`… (see [guideposts](https://taginfo.openstreetmap.org/keys/guidepost#combinations))
